pontoon/base/views.py

```diff
@@ -98,6 +98,30 @@ def translate(request, locale, slug, part):
     })
 
 
+# LOCALE-AGNOSTIC VIEWS
+
+
+def locale_agnostic(request, slug, part):
+    """Language Agnostic view."""
+    user = request.user
+    project = get_object_or_404(Project.objects.available(), slug=slug)
+    
+    if user.is_authenticated():
+
+        locale = user.profile.custom_homepage:
```

Comment:
This looks like a syntax error. You can check if your code contains critical syntax error with 'pylama' command.
Travis-ci executes that command everytime you push code to github e.g.
https://travis-ci.org/mozilla/pontoon/builds/327031767#L802 there's a list of errors from this commit that should be fixed.
(I'm not sure if you're familliar with github/travis-ci)